packages/core/build/config.js

```diff
@@ -127,27 +128,46 @@ const copyConfig = base => {
 
 const generateSWConfig = is_release => ({
     cleanupOutdatedCaches: true,
-    exclude: [
-        /CNAME$/,
-        /index\.html$/,
-        /404\.html$/,
-        /^localstorage-sync\.html$/,
-        /\.map$/,
-        /sitemap\.xml$/,
-        /robots\.txt$/,
-        /manifest\.json$/,
-        /^public\/images\/favicons\//,
-        /^favicon\.ico$/,
-        /^apple-app-site-association/,
-        /^assetlinks.json/,
-        /^.well-known\//,
-        /^account\//,
-        /^js\/smartcharts\//,
-        /^bot\//,
-        /^media\//,
-        /^trader\//,
-        /^cashier\//,
-        /^js\/core\.[a-z_]*-json\./,
+    exclude: [/^(?!css).*/],
+    runtimeCaching: [
+        {
+            urlPattern: /public\/(images|sprites)\/(?!.*favicons).*$/,
+            handler: 'CacheFirst',
+            options: {
+                cacheName: 'assets',
+                expiration: {
+                    maxEntries: 15,
```

Comment:
Can we add `maxAgeSeconds` and set it to 1 day.

Reply:
maxEntries has been added to limit the number to files cached, at this very moment there is 16 image in common folder, 3 in header and 13 in sprites folder, so if user is mainly using only cashier and for some reason he just visits all our app this will end up caching all the above files, since he is mainly using cashier we might not need to do cache all and let workbox handle to keep most used 15 files . thinking of that maxEntries I had kept to 15, should we remove it ?